Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/MainWindow.cpp

```diff
@@ -2813,6 +2818,55 @@ namespace EMStudio
         }
     }
 
-} // namespace EMStudio
+    void MainWindow::UpdatePlugins(float timeDelta)
+    {
+        EMStudio::PluginManager* pluginManager = EMStudio::GetPluginManager();
+        if (!pluginManager)
+        {
+            return;
+        }
+
+        const size_t numPlugins = pluginManager->GetNumActivePlugins();
+        for (size_t i = 0; i < numPlugins; ++i)
+        {
+            EMStudio::EMStudioPlugin* plugin = pluginManager->GetActivePlugin(i);
+            plugin->ProcessFrame(timeDelta);
+        }
+    }
 
-#include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/moc_MainWindow.cpp>
+    void MainWindow::EnableUpdatingPlugins()
+    {
+        AZ::TickBus::Handler::BusConnect();
```

Comment:
The TickBus is only active when the editor is in focus, should this connect to the SystemTickBus instead to ensure this behavior happens even if the editor is not in the foreground?

Reply:
Good question, what do you think?
How are other windows behaving in this regard @NicholasVanSickle ?

Reply:
I think we should keep it on the tick bus. That is always the behavior from SystemComponent.cpp.
Don't think there is a reason to tick the animation editor when it's in background.

Reply:
Would that be the expected behavior of someone working on through DCC? if they switch into their authoring software, would they want to continue seeing their animation in the editor?

Reply:
We would still skip rendering when the window is not visible @amzn-rhhong, we'd only keep rendering when losing focus, like when you keep the animation editor open on another screen.

Reply:
Actually, if we switch over to the other bus, we might be impacted by vsync again - so that would be a no-go if that is the case. Let's see what @NicholasVanSickle will say on this :).